atomix/cluster/src/main/java/io/atomix/raft/storage/RaftStorage.java

```diff
@@ -389,7 +389,7 @@ public boolean isRetainStaleSnapshots() {
     private int maxEntrySize = DEFAULT_MAX_ENTRY_SIZE;
     private int maxEntriesPerSegment = DEFAULT_MAX_ENTRIES_PER_SEGMENT;
     private long freeDiskSpace = DEFAULT_FREE_DISK_SPACE;
-    private boolean flushOnCommit = DEFAULT_FLUSH_ON_COMMIT;
+    private boolean flushExplicitly = DEFAULT_FLUSH_ON_COMMIT;
```

Comment:
```suggestion
    private boolean flushExplicitly = DEFAULT_FLUSH_EXPLICITLY;
```

Reply:
Will do, but I've already learned not to commit such fixes directly from Github since it'll break the build :sweat_smile: